.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
+    "restructuredtext.confPath" : "doc/manual/de/conf.py",
+    "restructuredtext.builtDocumentationPath" : "out/de/0.10.0-dev/manual/",
```

Comment:
Initilially I thought about automatically updating this path when the version changes (via our build tool), but a dedicated path just for this preview is indeed the better option, as it it version independent and does not has to be changed ever.